### ISSUE TITLE
usdchecker NameError when checking an unsupported file

### ISSUE THREAD

**Issue by hinton-mgi** [PixarAnimationStudios/USD#1899]
Rather than gracefully printing a descriptive error, usdchecker fails with a NameError when checking an unsupported file.
For example:

```
> usdchecker CMakeLists.txt
Traceback (most recent call last):
  File "D:\Testing\USD-sdk\bin\usdchecker", line 149, in <module>
    sys.exit(main())
  File "D:\Testing\USD-sdk\bin\usdchecker", line 118, in main
    checker.CheckCompliance(inputFile)
  File "D:\Testing\USD-sdk\lib\python\pxr\UsdUtils\complianceChecker.py", line 953, in CheckCompliance
    _AddError("Cannot open file '%s' on a USD stage." % args.inputFile)
NameError: name '_AddError' is not defined
```

**Comment by spiffmon** [comment]
Thanks, @hinton-mgi .. simple case of a missing `self.` referential that there was no compiler to catch.

**Comment by sunyab** [comment]
Filed as internal issue #USD-7422

**Comment by tallytalwar** [comment]
This is fixed by #2429 and the associated commit: 2a943aac82443e77585ef39741a425eb44ee1980 is available in 23.11.
Thanks
